api/src/main/java/io/opencensus/stats/NoopStats.java

```diff
@@ -180,5 +183,18 @@ public ViewData getView(View.Name name) {
         }
       }
     }
+
+    @Override
+    public Set<View> getAllExportedViews() {
```

Comment:
Why does not return empty set for no-op?  I think this is more like "testing" implementation. If you need a testing implementation you should add one in the testing package, and keep the no-op as no-op implementation.